### PR TITLE
Basic Admin order creation scenarios

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: features/creating_order_with_online_payment.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@managing_orders
+Feature: Creating order with online payment
+    In order to have order placed by Admin and paid by Customer
```
Comment (CoderMaggie) [inline features/creating_order_with_online_payment.feature]: it is not a real benefit ;)

Comment (Zales0123) [reply features/creating_order_with_online_payment.feature]: Alternative suggestion is welcomed 😄

Comment (CoderMaggie) [inline features/creating_order_with_online_payment.feature]: `In order to place order in he name of a Customer and forward the payment to them`? wdyt?

File: features/creating_simple_order.feature [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@managing_orders
+Feature: Creating simple order
+    In order to have order placed without Customer intervention
```
Comment (CoderMaggie) [inline features/creating_simple_order.feature]: `In order to place an order in the name of a Customer`?

**Pull request merged**